rotkehlchen/chain/ethereum/modules/ens/decoder.py

```diff
@@ -71,19 +79,27 @@ def _decode_ens_registrar_event(self, context: DecoderContext) -> DecodingOutput
 
     def _decode_name_registered(self, context: DecoderContext) -> DecodingOutput:
         try:
-            _, decoded_data = decode_event_data_abi_str(context.tx_log, NAME_REGISTERED_ABI)
+            _, decoded_data = decode_event_data_abi_str(
+                context.tx_log,
+                NAME_REGISTERED_1_ABI if context.tx_log.topics[0] == NAME_REGISTERED_1 else NAME_REGISTERED_2_ABI,  # noqa: E501
+            )
         except DeserializationError as e:
             log.debug(f'Failed to decode ENS name registered event due to {str(e)}')
             return DEFAULT_DECODING_OUTPUT
 
         name = decoded_data[0]
-        amount = from_wei(decoded_data[1])
-        expires = decoded_data[2]
+        if context.tx_log.topics[0] == NAME_REGISTERED_1:
+            amount = from_wei(decoded_data[1])
+            expires = decoded_data[2]
+        else:
+            amount = from_wei(decoded_data[1] + decoded_data[2])
```

Comment:
why is now the sum of this two places? Could you add a comment with the change that they have made to make it clearer for the reader?